modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -19383,7 +19383,7 @@ this-publication-contains-conflicting-changes-that-must-be-manually-resolved-bef
 this-publication-contains-unapproved-changes=This publication contains unapproved changes.
 this-publication-contains-unapproved-changes-that-must-be-approved-before-publishing=This publication contains unapproved changes that must be approved before publishing.
 this-publication-no-longer-exists=This publication no longer exists.
-this-publication-was-created-on-a-previous-liferay-version.-you-cannot-publish,-revert,-or-make-additional-changes=This publication was created on a previous Liferay version. You cannot publish, revert, or make additional changes. You can only view or delete it.
+this-publication-was-created-on-a-previous-liferay-version.-you-cannot-publish,-revert,-or-make-additional-changes=This publication was created on a previous Liferay version. You cannot publish, revert, or make additional changes. You can only view, delete it or move its changes to another publication.
```

Comment:
The last sentence should be `You can only view, delete, or move changes to another publication.`